src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx

```diff
@@ -1,56 +1,39 @@
 import { Button, Clickable, Flex, ModalDialog, Text } from "@artsy/palette"
-import { toTitleCase } from "@artsy/to-title-case"
+
+import { MyCollectionArtworkSWASectionSubmitted_submissionState$key } from "__generated__/MyCollectionArtworkSWASectionSubmitted_submissionState.graphql"
 
 import { useState } from "react"
+import { graphql, useFragment } from "react-relay"
 import { RouterLink } from "System/Router/RouterLink"
 import { Media } from "Utils/Responsive"
 
-// TODO:- We are using displayText for Statuses for now. Consider changing the logic when proper statuses are made available on Metaphysics.
-// See https://artsyproduct.atlassian.net/browse/SWA-217
-// same as on Eigen
-const STATUSES: {
-  [key: string]: { color: string; text: string; description?: string }
-} = {
-  "submission in progress": {
-    color: "yellow150",
-    text: "In Progress",
-    description: "The artwork is being reviewed or is in the sale process.",
-  },
-  "submission evaluated": {
-    color: "orange150",
-    text: "Evaluation Complete",
-    description:
-      "Our specialists have reviewed this submission and determined that we do not currently have a market for it.",
-  },
-  sold: { color: "black100", text: "Artwork Sold" },
-}
-
 interface Props {
-  displayText: string
+  artwork: MyCollectionArtworkSWASectionSubmitted_submissionState$key
 }
 
 export const MyCollectionArtworkSWASectionSubmitted: React.FC<Props> = ({
-  displayText,
+  artwork,
 }) => {
   const [
     isSubmissionStatusModalOpen,
     setIsSubmissionStatusModalOpen,
   ] = useState(false)
 
-  if (!Boolean(displayText)) {
-    return null
-  }
+  const { consignmentSubmission } = useFragment(
+    submissionStateFragment,
+    artwork
+  )
+  if (!consignmentSubmission) return null
 
-  const article =
-    "https://support.artsy.net/s/topic/0TO3b000000UevOGAS/sell-with-artsy"
+  const { state, stateLabel, stateHelpMessage } = consignmentSubmission
+  if (!state) return null
 
-  const approvedDisplayText = STATUSES[displayText.toLowerCase()]?.text
-  const statusDescription =
-    STATUSES[displayText.toLowerCase()]?.description || ""
+  let stateLabelColor = "yellow150"
+  if (["APPROVED", "REJECTED", "CLOSED", "PUBLISHED"].includes(state))
+    stateLabelColor = "orange150"
```

Comment:
@joeyAghion you said that there are no submissions in "sold" state anymore - this we the only one with `black100` color. It's ok that I do not support it anymore, right?

Reply:
Correct. That isn't a [valid submission state](https://github.com/artsy/convection/blob/55087cd8e2433de779cae9655cdd9fcdd7415d22/app/models/submission.rb#L25), but [came from](https://github.com/artsy/metaphysics/blob/04f67e059726724c114f0efd7f54359e1152cdc0/src/schema/v2/artwork/artworkConsignmentSubmissionType.ts#L40) the `consigned_partner_submission` that hasn't been in use recently.